deegree-services/deegree-webservices-handbook/src/main/asciidoc/renderstyles.adoc

```diff
@@ -874,7 +874,7 @@ The first parameter is the hatching angle, the second is the line spacing in the
 </PolygonSymbolizer>
 ----
 TIP: For of the shelf hates, which will create nice results, use the mark symbol `shape://slash`, `shape://backslash`
-or `shape://times` for  45°, `shape://horline` for 0° and `shape://vertline` for 90° hatches.
+or `shape://times` for  45�, `shape://horline` for 0� and `shape://vertline` for 90� hatches.
```

Comment:
Please review the encoding of this file. The special character for deegree is not displayed correctly.

Reply:
The encoding was fixed in https://github.com/deegree/deegree3/pull/1407/commits/ec252f76e0908acc3bc94ceb3cc28419c74c2376